client/galaxy/scripts/components/InteractiveTools/InteractiveTools.test.js

```diff
@@ -23,51 +24,40 @@ describe("ToolsView/ToolsView.vue", () => {
                 },
             },
         });
-        emitted = wrapper.emitted();
         axiosMock.onGet("/api/entry_points?running=true").reply(200, testInteractiveToolsResponse);
         axiosMock.onPost("/interactivetool/list").reply(200, { status: "ok", message: "ok" });
-        await Vue.nextTick();
-        await Vue.nextTick();
-        await Vue.nextTick();
+        await flushPromises();
```

Comment:
This is exciting - I use this repeated nextTick hack all over. Thanks for cleaning this up.

Reply:
@jmchilton Yeah, I was going to make a pass swapping them across the board separately.  nextTick is fine for waiting on internal component rendering, but for triggered events and axios, etc., we should just use flushPromises.